### ISSUE TITLE
Missing spacing in "Alt+Shift+R" shortcut in A11y Keyboard Shortcuts

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/energy-skate-park#593]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox

**Problem description**
From https://github.com/phetsims/qa/issues/1395

In the A11y view, add a space for the "Restart skater with **AltplusShift** plus R key"   for the _From Anywhere in Screen_ section in Keyboard Shortcuts.

**Visuals**

<img width="447" height="151" alt="Image" src="https://github.com/user-attachments/assets/5d1a7324-f4ea-4d64-950b-31943714e94a" />

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Energy Skate Park‬
URL: https://phet-dev.colorado.edu/html/energy-skate-park/1.5.0-rc.4/phet/energy-skate-park_all_phet.html
Version: 1.5.0-rc.4 2026-05-02 03:41:10 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/147.0.0.0 Safari/537.36
Language: en-US
Window: 1440x731
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by samreid** [comment]
This code trims strings during a build (before adding the directionality markers): https://github.com/phetsims/totality/blob/3dfc6aa911a660607068594af5dd6bbd9aacc527/chipper/js/common/ChipperStringUtils.ts#L117-L119

Can we add a space like so? It works correctly after a build.

<details>

```diff

Subject: [PATCH] Update text, see https://github.com/phetsims/energy-skate-park/issues/575
---
Index: scenery-phet/js/keyboard/help/HotkeyDescriptionBuilder.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scenery-phet/js/keyboard/help/HotkeyDescriptionBuilder.ts b/scenery-phet/js/keyboard/help/HotkeyDescriptionBuilder.ts
--- a/scenery-phet/js/keyboard/help/HotkeyDescriptionBuilder.ts	(revision d8553d84189b075fef261c98ee5df6607f317a6f)
+++ b/scenery-phet/js/keyboard/help/HotkeyDescriptionBuilder.ts	(date 1778199536351)
@@ -190,7 +190,9 @@
     const sorted = HotkeySetDefinitions.sortModifiers( modifiers );
     const labels = sorted.map( modifier => HotkeyDescriptionBuilder.describeModifier( modifier ) );
     const spacePlusSpaceStringProperty = SceneryPhetFluent.a11y.keyboard.helpPatterns.spacePlusSpaceStringProperty;
-    return labels.join( spacePlusSpaceStringProperty.value );
+
+    // Built strings are trimmed by ChipperStringUtils, so do not rely on leading/trailing spaces in this string.
+    return labels.join( ` ${spacePlusSpaceStringProperty.value.trim()} ` );
   }
 
   /**


```

</details>

Are there other places that will fail due to this trim? Could VoiceOver pronounce it correctly anyways because it still retains the directionality markers? I suspect not but did not know how to get VoiceOver to read through the keyboard help dialog.

@jessegreenberg how would you like to proceed?
